Assets/StreamingAssets/Data/Inventory.xml

```diff
@@ -2,20 +2,32 @@
 <Inventories>
   <Inventory objectType="Raw Iron">
```

Comment:
Once again, I followed the already setup format of the file. And this isn't relevant as we will probably go away from XML. This one doesn't matter for me. 

Feel free to insist if you really want me to change it. I just need the feature for other PR personally and I know the file format will change soon.